packages/mui-material/src/Fab/Fab.js

```diff
@@ -52,6 +52,7 @@ const FabRoot = styled(ButtonBase, {
     minWidth: 0,
     width: 56,
     height: 56,
+    zIndex: 1050,
```

Comment:
The value should not be hardcoded. A new field called `fab` should be created in packages\mui-material\src\styles\zIndex.js and then used similarly to other zIndex values.

Reply:
I see. Will update that later.